cardano-testnet/cardano-testnet.cabal

```
@@ -198,8 +198,10 @@ test-suite cardano-testnet-test
                       , cardano-cli
                       , cardano-crypto-class
                       , cardano-ledger-conway
+                      , cardano-ledger-core
                       , cardano-ledger-shelley
                       , cardano-node
+                      , cardano-strict-containers
```

```suggestion
                      , cardano-strict-containers ^>= 0.1
```

To have the same bound as [the existing occurrence](https://github.com/IntersectMBO/cardano-node/blob/master/bench/locli/locli.cabal#L146) in this repo?